src/components/views/investors/SignupLanding.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Signup from './SignUp.jsx';
+import LoginSideBar from '../../common/InvestorsLoginSidebar';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../common/InvestorsLoginSidebar"](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)